doc/api.rst

```diff
@@ -317,6 +319,31 @@ custom argument formatter class::
 .. doxygenclass:: fmt::arg_formatter
    :members:
 
+.. _ranges-api:
+
+Ranges and Tuple Formatting
+===========================
+
+The library also supports convenient formatting of ranges and tuples::
+
+  #include <fmt/ranges.h>
+
+  std::tuple<char, int, float> t{'a', 1, 2.0f};
+  // Prints a 1 2.0
+  fmt::print("{}", t);
+
+
+NOTE: currently, the overload of `fmt::join` for iterables exists in the main
```

Comment:
Inline code is double-tick delimited in RST, e.g. ``fmt::join``.

Reply:
Ah ok I'll fix that thanks